config/routes.rb

```diff
@@ -1,11 +1,12 @@
 IntergalacticBattleship::Application.routes.draw do
+  devise_for :users
   root "strikes#index"
   resources :strikes, only: [:new, :create]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'static_pages#index'
```

Comment:
@kenmazaika I'm thinking it's an issue with devise and the two root pages going on. Whenever I put the second root back then I get an error message of 

> /home/vagrant/.rbenv/versions/2.0.0-p353/lib/ruby/gems/2.0.0/gems/actionpack-4.0.1/lib/action_dispatch/routing/route_set.rb:434:in `add_route': Invalid r         oute name, already in use: 'root'  (ArgumentError)
You may have defined two routes with the same name using the `:as` option, or you may be overriding a route already defined by a resource with the same n         aming. For the latter, you can restrict the routes created with `resources` as explained here:
http://guides.rubyonrails.org/routing.html#restricting-the-routes-created

But when I comment out the second root I don't get that message. But I get why the other error is popping up because there is an spec test written for that second root page and a controller.